src/osgWrappers/serializers/osg/Texture.cpp

```diff
@@ -245,10 +245,11 @@ REGISTER_OBJECT_WRAPPER( Texture,
         ADD_ENUM_VALUE( ALWAYS );
     END_ENUM_SERIALIZER();  // _shadow_compare_func
 
-    BEGIN_ENUM_SERIALIZER( ShadowTextureMode, LUMINANCE );
+    BEGIN_ENUM_SERIALIZER( ShadowTextureMode, NONE );
```

Comment:
This change will break backwards compatibility.

Reply:
Accounting that in most of the case ShadowTextureMode is left untouched by the user..it may not have be written in files(default) so changing default value in serializer may fix inconsistancies...
Am I right?
And in the case user have setted it exiplicitly there's no issue